webapp/channels/src/components/emoji_picker/emoji_picker_tabs.tsx

```diff
@@ -170,6 +170,9 @@ export default class EmojiPickerTabs extends PureComponent<Props, State> {
                 className={classNames('a11y__popup', 'emoji-picker', 'emoji-picker--single', {
                     bottom: this.props.placement === 'bottom',
                 })}
+                role='dialog'
+                aria-label='Emoji picker'
```

Comment:
```suggestion
                aria-labelledby='Emoji picker'
```

Reply:
The emoji picker did not have a visible title, so I added an aria-label instead aria-labelledby.